java/integTest/src/test/java/glide/LoggerTests.java

```diff
@@ -62,25 +67,30 @@ public void log_to_file() {
         Logger.log(Logger.Level.DEBUG, debugIdentifier, () -> debugMessage);
         Logger.log(Logger.Level.TRACE, traceIdentifier, () -> traceMessage);
 
+        // Initialize a new logger to force closing of existing files
+        Logger.setLoggerConfig(Logger.Level.DEFAULT, "dummy.txt");
```

Comment:
Should we delete this if it exists as well? Not sure if it's lazily created on the first log()

Reply:
I'm deleting this and the log folder now